src/src/service/model.d.ts

```diff
@@ -1,16 +1,16 @@
 
```

Comment:
Do not commit those changes in this PR - they are formatting changes in file otherwise untouched by PR. Otherwise this will obscure git history.